### PR TITLE
remove usage of section insets to calculations that caused issues wit…

### PR DESCRIPTION
**Pull request by paul-moon**
…h the tiles

### CODE REVIEW

**Comment by paul-moon** [comment]
+1 to myself

**Comment by edgargigaramirez** [comment]
+1

File: RFQuiltLayout/RFQuiltLayout.m [hunk 1]
```diff
@@ -466,11 +467,11 @@ - (CGRect) frameForIndexPath:(NSIndexPath*)indexPath {
...
     CGPoint position = [self positionForIndexPath:indexPath];
     CGSize elementSize = [self getBlockSizeForItemAtIndexPath:indexPath];
-	UIEdgeInsets sectionInset = [self sectionInsetForSection:indexPath.section];
+    UIEdgeInsets sectionInset = [self sectionInsetForSection:indexPath.section];
     
     if (isVert) {
-        return CGRectMake(position.x*self.itemBlockSize.width + sectionInset.left,
-                          position.y*self.itemBlockSize.height + ((indexPath.section+1) * (sectionInset.top + self.headerReferenceSize.height)) + (indexPath.section * self.footerReferenceSize.height) + (indexPath.section * sectionInset.bottom),
+        return CGRectMake(position.x*self.itemBlockSize.width,
+                          position.y*self.itemBlockSize.height + ((indexPath.section+1) * (self.headerReferenceSize.height)) + (indexPath.section * self.footerReferenceSize.height) + (indexPath.section),
```
Comment (quinnmackenzie) [inline RFQuiltLayout/RFQuiltLayout.m]: This is the only actual content change other than the other line at the bottom, right?

File: RFQuiltLayout/RFQuiltLayout.m [hunk 2]
```diff
@@ -500,7 +501,7 @@ - (CGSize)getBlockSizeForItemAtIndexPath:(NSIndexPath *)indexPath
 - (int) restrictedDimensionBlockSize {
     BOOL isVert = self.scrollDirection == UICollectionViewScrollDirectionVertical;
     
-    int size = isVert? (self.collectionView.frame.size.width - (self.sectionInset.right + self.sectionInset.left)) / self.itemBlockSize.width : (self.collectionView.frame.size.height - (self.sectionInset.top + self.sectionInset.bottom)) / self.itemBlockSize.height;
+    int size = isVert? self.collectionView.frame.size.width / self.itemBlockSize.width : self.collectionView.frame.size.height / self.itemBlockSize.height;
```
Comment (quinnmackenzie) [inline RFQuiltLayout/RFQuiltLayout.m]: This is the one of the only actual content changes, right?

File: RFQuiltLayout/RFQuiltLayout.m (hunk 1 above)
Comment (paul-moon) [inline RFQuiltLayout/RFQuiltLayout.m]: yeah pretty much.  I copied + pasted from the word i did in the original pod file and the other changes showed up.

File: RFQuiltLayout/RFQuiltLayout.m (hunk 2 above)
Comment (paul-moon) [inline RFQuiltLayout/RFQuiltLayout.m]: yeah pretty much.  I copied + pasted from the word i did in the original pod file and the other changes showed up.

**Comment by quinnmackenzie** [comment]
Looks good, although I'm wondering, did you try using the original code with the section insets set to 0 first?  Or is there a reason they have to be non-zero but still have the behavior in restrictedDimensionBlockSize and frameForIndexPath that changed here?

**Comment by paul-moon** [comment]
@quinnmackenzie  to be honest, i didn't look too deeply into the other methods.  I was kind of rushing to make it work at the moment.  I have no problem going back and cleaning things up time permitting.

**Comment by quinnmackenzie** [comment]
@paul-moon Basically I was just wondering if you tried setting the section insets to 0 while using the original library.  Because the only code changes here are equivalent to the original library behavior in the case where the section insets are 0.

**Comment by paul-moon** [comment]
@quinnmackenzie that wasn't the only change actually.  There were changes to added like referenceHeaderSize and etc.  

Two commits that some some changes

https://github.com/HyattDXLab/RFQuiltLayout/commit/2e213f5fb7291672a886028057b141b48366e409

https://github.com/HyattDXLab/RFQuiltLayout/commit/1496ece9b30b987920a8a8317329795659fac487

**Comment by quinnmackenzie** [comment]
@paul-moon Oh, ok.  So which are we actually using in our project?  The fork or the fork of a fork?

**Comment by paul-moon** [comment]
@quinnmackenzie The fork of a fork.  Confusing I know.  I wish I stumbled upon the issue earlier so I had more time to investigate it.

**Comment by quinnmackenzie** [comment]
@paul-moon Ok, sounds good.  It didn't look like that other fork was really under active development anymore, so probably makes sense to have our own version of it.

**Pull request merged**